cpp/models/ode_secirvvs/model.h

```diff
@@ -242,8 +242,7 @@ class Model
                         pop[ISyNCj] + pop[SPIj] + pop[EPIj] + pop[INSPIj] + pop[ISyPIj] + pop[ISevPIj] + pop[ICrPIj] +
                         pop[INSPICj] + pop[ISyPICj] + pop[SIIj] + pop[EIIj] + pop[INSIIj] + pop[ISyIIj] + pop[ISevIIj] +
                         pop[ICrIIj] + pop[INSIICj] + pop[ISyIICj];
-
-                FP divNj = 1.0 / Nj; // precompute 1.0/Nj
+                const FP divNj = (Nj < 1e-12) ? 0.0 : 1.0 / Nj;
```

Comment:
I think this is a good fix, since it stays in the model definition. The addition of the tests is also nice.

The only question I have is: why 1e-12?
This number may depend on ScalarType/FP, so should we use a global constant? (e.g. the accuracy of double is at e-16, float only at e-8)